src/data/var/acceptData.js

```diff
@@ -13,7 +13,6 @@ return function( owner ) {
 	//    - Node.DOCUMENT_NODE
 	//  - Object
 	//    - Any
-	/* jshint -W018 */
```

Comment:
http://eslint.org/docs/rules/no-negated-in-lhs handles one of the cases W018 would apply to, but not this one...